src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/Extensions/JsonConverterCollectionExtensions.cs

```diff
@@ -3,34 +3,31 @@
 
 using System;
 using System.Collections.Generic;
-using Microsoft.CodeAnalysis.Razor.Serialization;
+using Microsoft.AspNetCore.Razor.ProjectEngineHost.Serialization;
 using Microsoft.CodeAnalysis.Razor.Workspaces.Serialization;
 using Newtonsoft.Json;
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.Common.Extensions;
 
 internal static class JsonConverterCollectionExtensions
 {
-    public static readonly IReadOnlyList<JsonConverter> RazorConverters = new List<JsonConverter>()
+    private static readonly IReadOnlyList<JsonConverter> s_razorConverters = new List<JsonConverter>()
```

Comment:
Since you're making this private, how about changing the field type to a `List<JsonConverter>` so you can do an allocation-free `foreach` below in `RegisterRazorConverters`?